components/vf-tabs/vf-tabs.scss

```diff
@@ -13,6 +13,31 @@ $vf-tabs__list--active-border--color: set-ui-color(vf-ui-color--white);
   border-bottom: 1px solid $vf-tabs__list-border--color;
   margin: 0;
   padding: 0 0 0px 16px;
+  position: relative;
+
+  // @media (min-width: $vf-breakpoint--xl) {
```

Comment:
From reading your PR description, it sounds like this shouldn't be commented out?

If you did intend to use it, one thing: the tab line could flow into other content. Example: if `vf-tabs` is inside a `.vf-grid__col-2`. I suppose there might be other times, like an EMBL Grid container, but that's the only one that comes to mind; screenshot:

![image](https://user-images.githubusercontent.com/928100/65173742-c1310e80-da4f-11e9-8332-8860dd479838.png)

Reply:
Maybe the easiest way around is to do `vf-tabs vf-tabs--full-width` or smthn.

Reply:
I commented this out - as it breaks the overflow of the page. It'd be neat to have - but it's probably more it's own little PR